dev/tests/integration/testsuite/Magento/Catalog/Model/ImageUploaderTest.php

```diff
@@ -43,16 +48,22 @@ protected function setUp()
         /** @var \Magento\Framework\Filesystem $filesystem */
         $this->filesystem = $this->objectManager->get(\Magento\Framework\Filesystem::class);
         $this->mediaDirectory = $this->filesystem->getDirectoryWrite(DirectoryList::MEDIA);
-        /** @var $uploader \Magento\MediaStorage\Model\File\Uploader */
-        $this->imageUploader = $this->objectManager->create(
-            \Magento\Catalog\Model\ImageUploader::class,
+        $this->configProvider = $this->objectManager->create(
```

Comment:
Not too sure if this is the desired way to do this or if I can inject the virtual type from the di?